src/router/routes.js

```diff
@@ -1,37 +1,43 @@
-	{ path: '/build/docs', component: () => import('@/pages/Build/DocsPage') },
-	{ path: '/build/getting-started', component: () => import('@/pages/Build/GettingStartedPage') },
-	{ path: '/build/code-of-conduct', component: () => import('@/pages/Build/CodeOfConductPage') },
-	{ path: '/build/data-snapshots', component: () => import('@/pages/Build/DataPage') },
 	{
 		path: '/algolia-vue',
 		component: () => import('@/pages/AlgoliaPOC'),
 		children: [
 			{ path: '*', component: () => import('@/pages/AlgoliaPOC') },
 		]
 	},
+	{ path: '/build', component: () => import('@/pages/Build/BuildPage') },
+	{ path: '/build/code-of-conduct', component: () => import('@/pages/Build/CodeOfConductPage') },
+	{ path: '/build/data-snapshots', component: () => import('@/pages/Build/DataPage') },
+	{ path: '/build/docs', component: () => import('@/pages/Build/DocsPage') },
+	{ path: '/build/getting-started', component: () => import('@/pages/Build/GettingStartedPage') },
+	{ path: '/build/research', component: () => import('@/pages/Build/Research') },
 	{ path: '/build/terms-of-service', component: () => import('@/pages/Build/TermsOfService') },
+	{ path: '/checkout', component: () => import('@/pages/Checkout/CheckoutPage') },
+	{ path: '/component-demo', component: () => import('@/pages/ComponentDemo/ComponentDemo') },
+	{ path: '/error', component: () => import('@/pages/Error') },
+	{ path: '/funded/:id', component: () => import('@/pages/BorrowerProfile/fundedBorrowerProfile') },
 	{ path: '/join-team', component: () => import('@/pages/LoginAndRegister/JoinTeam') },
+	{ path: '/kiva-app-components', component: () => import('@/pages/ComponentDemo/ComponentDemoKivaApp') },
+	{ path: '/legal/promo-terms-of-use-february-2020', component: () => import('@/pages/Legal/PromoTermsOfUse') },
+	{ path: '/legal/terms-of-use-february-2020', component: () => import('@/pages/Legal/TermsOfUse') },
+	{ path: '/lend-by-category', component: () => import('@/pages/Lend/LendByCategoryPage') },
+	{ path: '/lend-by-category/:category', component: () => import('@/pages/Lend/LoanChannelCategoryPage') },
+	{ path: '/lend-vue', component: () => import('@/pages/Lend/LendPage') },
+	{ path: '/lend/filter', component: () => import('@/pages/Lend/Filter/LendFilterPage') },
+	{ path: '/page-two', component: () => import('@/pages/PageTwo') },
+	{ path: '/portfolio/lending-stats', component: () => import('@/pages/LendingStats/LendingStatsPage') },
+	{ path: '/process-browser-auth', component: () => import('@/pages/ProcessBrowserAuth') },
+	{ path: '/protocol', component: () => import('@/pages/Protocol') },
 	{ path: '/register/social', component: () => import('@/pages/LoginAndRegister/RegisterSocial') },
-	{ path: '/build/research', component: () => import('@/pages/Build/Research') },
-	{ path: '/error', component: () => import('@/pages/Error') },
+	{ path: '/settings/autolending', component: () => import('@/pages/Autolending/AutolendingPage')	},
+	{
+		path: '/settings/autolending/opt-out',
```

Comment:
This is the new route. The rest of the changes here are just alphabetizing the routes to make them a little more organized